docs/docs/guides/create_firehose.md

```diff
@@ -140,4 +142,11 @@ _**Note:**_ [_**DATABASE**_](../sinks/influxdb-sink.md#sink_influx_db_name) _**a
 
 - it requires the following environment  [variables](https://github.com/goto/depot/blob/main/docs/reference/configuration/bigtable.md) ,which are required by Depot library, to be set along with the generic firehose variables.
 
+## Create an HTTPV2 Sink
+
+- HttpV2 Sink is implemented in Firehose using the Http sink connector implementation in Depot library. For details on all the features supported by HttpV2 Sink, please refer the Depot documentation [here](https://github.com/goto/depot/blob/main/docs/sinks/http.md).
```

Comment:
@sumitaich1998 this doc doesn't exist. Can you update the correct link before this gets merged?

<img width="1176" alt="image" src="https://github.com/goto/firehose/assets/6890568/0b078123-d835-491e-83ca-6745bb58cb57">